docs/changes/visualization/17243.feature.rst

```diff
@@ -0,0 +1,3 @@
+WCSAxes will now automatically select the fastest varying coordinate to show
+along each axis, rather than picking them in the order they are listed in the
+WCS.
```

Comment:
```suggestion
WCSAxes will now select which axis to draw which tick labels and axis labels on based on the number of drawn tick labels, rather than picking them in the order they are listed in the WCS. This means that axes may be swapped in comparison with previous versions of Astropy by default.
```